airsonic-main/src/test/java/org/airsonic/player/service/search/AnalyzerFactoryTestCase.java

```diff
@@ -0,0 +1,574 @@
+
```

Comment:
Please use 4 space indentation. @jvoisin please add this to the list in #1121

Reply:
I revised that according to your advice.